src/FlowAD/Types/type_utils.jl

```diff
@@ -40,8 +40,8 @@ function setup_well_model(M::jutulModel{D, T}, f::Union{jutulForce{D, T}, jutulV
     model, parameters = setup_reservoir_model(domain, sys, wells = Is)
```

Comment:
In JutulDarcy, `setup_reservoir_model()`  calls `setup_parameters()`, so we could change `setup_reservoir_model()` to pass the kwargs to `setup_parameters()`.

Then we would specify the parameters as below:
```julia
    parameters = Dict(:Reservoir=>Dict(:PhaseViscosities => vcat(visCO2 * ones(prod(M.n))', visH2O * ones(prod(M.n))')))
    model, parameters = setup_reservoir_model(domain, sys, wells = Is, parameters...)
```

I'm not sure if using the kwargs for that would badly affect anything in JutulDarcy. We could ask @moyner's opinion on this change.